dbms/src/Storages/DeltaMerge/StoragePool.cpp

```diff
@@ -211,13 +299,99 @@ StoragePool::~StoragePool()
 
 void StoragePool::enableGC()
```

Comment:
Merge `enableGC` into `restore`?

Reply:
Separate them seems better which make the logic more clear although it's useless for V3.